ocean_lib/test/test_config_provider.py

```diff
@@ -20,14 +20,14 @@ def test_set_config():
 
 def test_metadataCacheUri_version():
     """Tests that the Aquarius URL can use the custom metadataCacheUri key."""
-    config_dict = {"metadataCacheUri": "http://ItWorked.com", "network": "rinkeby"}
+    config_dict = {"metadataCacheUri": "http://ItWorked.com", "network": "ganache"}
```

Comment:
why change the network name here?

Reply:
"rinkeby" caused a TypeError when constructing the OceanPool object at `ocean.py:120`.  `ocean_address` was `None` because the `~/.ocean/.../address.json` created by barge only contains "development" addresses, not "rinkeby" addresses.